internal/wasm/sdk/internal/wasm/vm.go

```diff
@@ -245,7 +245,7 @@ func (i *VM) Eval(ctx context.Context, entrypoint int32, input *interface{}, met
 	}
 
 	if input != nil {
-		inputAddr, err := i.toRegoJSON(*input, false)
+		inputAddr, err := i.toRegoJSON(*input, true)
```

Comment:
It looks like all of the callers pass `true` now. Unless we have use cases for NOT freeing the serialization buffer, let's just remove the param and always free.

Reply:
I think I'm missing context on why we're freeing the raw bytes here--since the heap ptr is reset on every `eval()` I'm guessing we're not doing this guard against leaks...or are we?

Reply:
You are right, the free is not required at this point, since we anyway reset the heap in the beginning of each eval. I was thinking of having the free here for clarity, but it's not really buying anything much but extra CPU cycles. I changed it back for now.